staking-pool/README.md

```diff
@@ -298,3 +298,117 @@ pub fn update_reward_fee_fraction(&mut self, reward_fee_fraction: RewardFeeFract
 /// behalf of this staking pool.
 pub fn vote(&mut self, voting_account_id: AccountId, proposal_id: ProposalId) -> Promise;
 ```
+
+## Migrating from an existing validator or contract
+This provides instructions to migrate your staked validator or a validator contract to the new contract for Stakewars2
+
+Upgrade to the latest near-shell:
+```bash
+npm install -g near-shell
+```
+Login:
+```bash
+#If not logged into the browser recover your account with the seed phrase first
+export NEAR_ENV=betanet
+near login
+```
+
+Unstake and Withdraw:
+```bash
+#If you staked to your validator unstake, there is no withdraw
+near stake nearkat.betanet <staking public key> 0
+
+#If you staked to a contract
+near call my_validator unstake '{"amount": "100000000000000000000000000"}' --accountId user1
```

Comment:
Add how to get the current amount from the contract:
`near view my_validator get_account_staked_balance '{"account_id": "user1"}'`